src/scenes/ProgressReports/EditForm/Steps/SubmitReportStep/ConfirmVariantDialog.tsx

```diff
@@ -0,0 +1,140 @@
+  const { session } = useSession();
+  const roles = new Set(session?.roles.value);
+  const hasReportProgressFile = !!reportFile.value;
+  const projectMgmtHeader = !hasReportProgressFile && reportFile.canEdit;
+
+  const handleConfirmDialog = () => {
+    onContinue();
+  };
+
+  const handleCloseDialog = () => {
+    onCancel();
+  };
+
+  const getEmptyResponsesForRole = (
+    list: PromptResponseListFragment,
+    roles: Set<Role>
+  ) => {
+    if (list.items.length === 0) return [];
+    const emptyResponsesByRole = list.items[0]?.responses.filter((item) => {
+      return item.variant.responsibleRole
+        ? roles.has(item.variant.responsibleRole) &&
+            !item.response.value &&
+            item.response.canEdit
+        : false;
+    });
+    return emptyResponsesByRole ? emptyResponsesByRole : [];
+  };
+
+  const listItemsData = [
+    {
+      key: 'teamNews',
+      condition:
+        !report.teamNews.items.length ||
+        getEmptyResponsesForRole(report.teamNews, roles).length,
+      label: 'Team News',
+    },
+    {
+      key: 'story',
+      condition:
+        !report.communityStories.items.length ||
+        getEmptyResponsesForRole(report.communityStories, roles).length,
+      label: 'Story',
+    },
+  ];
+  const listItems = listItemsData
+    .filter((item) => item.condition)
+    .map((item) => <li key={item.key}>{item.label}</li>);
+
+  const investorConnectionHeader = listItems.length > 0;
+  if (!investorConnectionHeader && !projectMgmtHeader && !open) {
+    onUpdateConfirmStatus();
+  }
+
+  return (
+    <Dialog
+      open={(investorConnectionHeader || projectMgmtHeader) && open}
+      onClose={handleCloseDialog}
+      aria-labelledby="dialog-title"
+    >
+      <DialogTitle id="dialog-title">
+        Are you sure you want to submit report?
+      </DialogTitle>
+      <DialogContent dividers sx={{ ul: { paddingInlineStart: 3 } }}>
+        <Typography component="h3" variant="h4" paragraph>
+          The following was not updated or left blank:
+        </Typography>
+        {investorConnectionHeader && (
+          <>
+            <Typography
+              component="h3"
+              variant="h4"
+              id="investor-connection"
+              fontSize="0.80rem"
+            >
+              INVESTOR CONNECTION
+            </Typography>
+            <ul aria-labelledby="investor-connection">{listItems}</ul>
+          </>
+        )}
+        {projectMgmtHeader && (
+          <>
+            <Typography
+              component="h3"
+              variant="h4"
+              id="project-management"
+              fontSize="0.80rem"
+            >
+              PROJECT MANAGEMENT
+            </Typography>
+            <ul aria-labelledby="progress">
+              <li>Progress</li>
+            </ul>
+          </>
+        )}
+      </DialogContent>
+      <DialogActions>
+        <Button onClick={handleCloseDialog} variant="text" color="error">
```

Comment:
I know this was in the designs but I don't like this. Red buttons indicate a destructive action, like deleting a file. This cancel is not destructive.
```suggestion
        <Button onClick={handleCloseDialog} variant="text" color="secondary">
```